frontend/src/components/projectsList.tsx

```diff
@@ -1,123 +0,0 @@
-import { SystemStyleObject } from "@styled-system/css";
```

Comment:
these files are preserved in a separate branch already, so we can remove them from production code